bower.json

```diff
@@ -25,7 +25,6 @@
   ],
   "dependencies": {
     "jquery": "^1.11.1",
-    "ember-data": "1.0.0-beta.14.1",
```

Comment:
The npm package will inject the code the application and install the bower package automatically.

https://github.com/emberjs/data/blob/v1.0.0-beta.14.1/lib/ember-addon/blueprints/ember-data/index.js